src/oss/confluent.sh

```diff
@@ -1053,6 +1071,21 @@ connect_restart_command() {
     echo "Not implemented yet!"
 }
 
+acl_command() {
+    local service="${1}"
+    shift
+    case "${service}" in
+        schema-registry)
+                    if cli_file_exists "sr-acl-cli" ;
```

Comment:
It's pretty simple. Can be 
`if [[ -f "${confluent_bin}/sr-acl-cli" ]] ; then`

I see the potential for use with other services in the future, but I'm also usually in favor of introducing code or features when they are actually used. Up to you though.